Gemfile

```diff
@@ -7,7 +7,7 @@ source 'https://rubygems.org'
 
 gem 'rails', case SETTINGS[:rails]
              when '6.0'
-               '~> 6.0.3.1'
+               '~> 6.0.4.7'
```

Comment:
Would it be safer to update to 6.0.3.7
```suggestion
               '~> 6.0.3.7'
```

Reply:
We are on 6.0.3.7 (via packaging) already.